plugins/module_utils/consul.py

```diff
@@ -105,9 +126,10 @@ class _ConsulModule:
     """
 
     api_endpoint = None  # type: str
-    unique_identifier = None  # type: str
+    unique_identifiers = None  # type: list
```

Comment:
You need to add backwards compatibility for `unique_identifier`, since module_utils are part of the public API of this collection.

Best adjust the code to use `unique_identifier` if it's not `None`, and use `unique_identifiers` otherwise (or something similar so that modules from other collections that use this module_utils continue to function).